telcell/cell_identity.py

```diff
@@ -0,0 +1,295 @@
+import re
+from typing import Optional, Any
+
+RADIO_GSM = "GSM"
+RADIO_UMTS = "UMTS"
+RADIO_LTE = "LTE"
+RADIO_NR = "NR"
```

Comment:
Potential candidate for an enum maybe, even though it wouldn't change too much.
```suggestion
class Radio(Enum):
    GSM = "GSM"
    UMTS = "UMTS"
    LTE = "LTE"
    NR = "NR"
```